Firmware/MVP/BATTSCI.ino

```diff
@@ -61,10 +61,10 @@ inline uint8_t BATTSCI_writeByte(uint8_t data)
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 
-void BATTSCI_sendFrames(struct packetTypes METSCI_Packets, uint8_t stackVoltage, int16_t batteryCurrent_Amps)
+void BATTSCI_sendFrames(struct packetTypes METSCI_Packets, uint8_t stackVoltage, int16_t batteryCurrent_Amps, uint8_t desiredBehaviourFlag)
```

Comment:
Pass stackVoltage instead.  See notes later on (when you call BATTSCI_sendFrames)